CI.md

```diff
@@ -4,13 +4,14 @@
 
-| Jobs              | Description                                    | Python Versions      | Ansible Versions                                                    |
-| ----------------- | ---------------------------------------------- | -------------------- | ------------------------------------------------------------------- |
-| changelog         | Checks for the presence of Changelog fragments | 3.9                  | devel                                                               |
-| Linters           | Runs `black` and `flake8` on plugins and tests | 3.9                  | devel                                                               |
-| Sanity            | Runs ansible sanity checks                     | 3.8, 3.9, 3.10, 3.11 | Stable-2.12, 2.13, 2.14 (not on py 3.11), Stable-2.15+ (not on 3.8) |
-| Unit tests        | Executes the unit test cases                   | 3.9, 3.10            | Stable-2.12+                                                        |
-| Integration tests | Executes the integration test suites           | 3.9                  | devel                                                               |
+| Jobs              | Description                                                      | Python Versions      | Ansible Versions                                                    |
+| ----------------- | ---------------------------------------------------------------- | -------------------- | ------------------------------------------------------------------- |
+| changelog         | Checks for the presence of Changelog fragments                   | 3.9                  | devel                                                               |
+| Linters           | Runs `black` and `flake8` on plugins and tests                   | 3.9                  | devel                                                               |
+| Sanity            | Runs ansible sanity checks                                       | 3.8, 3.9, 3.10, 3.11 | Stable-2.12, 2.13, 2.14 (not on py 3.11), Stable-2.15+ (not on 3.8) |
+| Unit tests        | Executes the unit test cases                                     | 3.9, 3.10            | Stable-2.12+                                                        |
+| Integration tests | Executes the integration test suites                             | 3.9                  | devel                                                               |
+| Modules generator | Generates collection modules when config/modules.yaml is updated | 3.9                  | Stable-2.12                                                         |
```

Comment:
We can also re-generate the collection and do a release without necessarily adding any new modules but only bringing some changes from the Cloud Control API (e.g., new options added), fixing bugs or update a template hosted in the generator. I think it's probably better to rephrase the description.